scripts/lib/settings-window-capture/cdp.test.mjs

```
@@ -0,0 +1,243 @@
+  });
+
+  it("clears the timer when the socket rejects the send", async () => {
+    const { cdp, socket } = await connectFake({ commandTimeoutMs: 50 });
+    socket.send = () => {
+      throw new Error("socket is dead");
+    };
+
+    await expect(cdp.send("Page.enable")).rejects.toThrow(/socket is dead/);
+    // A leaked timer would keep the loop alive and later fire against a
+    // pending entry that should already be gone.
+    await new Promise((resolve) => setTimeout(resolve, 80));
```

ESLint

[error] 'setTimeout' is not defined.

(no-undef)